api/turing/validation/validator.go

```diff
@@ -101,22 +100,14 @@ func newExperimentConfigValidator(expSvc service.ExperimentsService) func(valida
 		case string(models.ExperimentEngineTypeNop):
 			return
 		case string(models.ExperimentEngineTypeLitmus), string(models.ExperimentEngineTypeXp):
```

Comment:
We still have some tech debt in this repo, explicitly mentioning the internal exp engines in many places (including unit tests). But the concerning part is this validation which will prevent creation of experiments for other new experiment engine plugins. Will add this cleanup to the backlog.